transaction-service/src/main/kotlin/ua/com/foxminded/restClient/mapper/TransactionMapper.kt

```diff
@@ -10,3 +13,33 @@ interface TransactionMapper {
     fun entityToDto(transaction: Transaction): TransactionDto
 
 }
+
+fun TransactionMapper.dtoToProto(
+    dto: TransactionDto
+): ProtoMessage.Transaction {
```

Comment:
```suggestion
fun TransactionDto.toProto(): ProtoMessage.Transaction {
```
then you won't need to inject your TransactionMapper